src/SFML/Window/Unix/JoystickImpl.cpp

```diff
@@ -315,10 +309,8 @@ unsigned int getUdevAttributeUint(udev_device* udevDevice, const std::string& at
     if (!udevDevice)
         return 0;
 
-    const char*  attribute = getUdevAttribute(udevDevice, attributeName);
-    unsigned int value     = 0;
-
-    if (attribute)
+    unsigned int value = 0;
```

Comment:
The variable `value` could be removed and simply return the result without assigning